network-api/networkapi/templates/donate/fragments/nav.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n primary_active_nav %}
```

Comment:
I mocked the existing nav code but simplified it quite a bit because donate nav is pretty bare and simple compared to the main site. I also converted all CSS/Bootstrap into Tailwind classes.

Reply:
Great! Thanks for doing the tailwind conversion @mmmavis, I know it must have been quite a lot of work 🙌